src/org/zaproxy/zap/extension/fieldenumeration/FieldEnumeration.java

```diff
@@ -0,0 +1,133 @@
+import org.parosproxy.paros.network.HttpMalformedHeaderException;
+import org.zaproxy.zap.view.messagecontainer.http.HttpMessageContainer;
+
+public class FieldEnumeration extends AbstractDialog {
+
+	private static final long serialVersionUID = 1L;
+	private static final Logger logger = Logger.getLogger(FieldEnumeration.class);	
+	
+//	private final ExtensionHistory extension;
+        private JPanel jPanel = new JPanel();
+	private JLabel printURL = new JLabel();	
+	private JButton buttonOK = new JButton("SUBMIT");
+	private ButtonGroup group = new ButtonGroup();
+	
+	private HistoryReference historyRef;
+	
+	/*public static void main(String args[]) {
+		new FieldEnumeration();
+	}*/
+	public FieldEnumeration() throws HeadlessException {
+		super();
+		initialize();
+	}
+	
+	public FieldEnumeration(Frame arg0, boolean arg1) throws HeadlessException {
+      		super(arg0, arg1);
+        	initialize();
+       }
+
+	
+	private void initialize() {
+//		this.setSize(600, 600);
+        this.setTitle(Constant.messages.getString("fieldenumeration.field.popup"));
+		this.setContentPane(getJPanel());
+		
+		/*Map<String, String> params = Model.getSingleton().getSession().getParams(historyRef, HtmlParameter.Type.Form);
+		for(Map.Entry<String, String> entry : params.entrySet()){
+			System.out.println(entry.getKey()+" :: "+entry.getValue());
+		}*/
+		
+		if (Model.getSingleton().getOptionsParam().getViewParam().getWmUiHandlingOption() == 0) {
+                	this.setSize(400, 400);
+	        }
+        	this.addWindowListener(new java.awt.event.WindowAdapter() {
+                @Override
+                	public void windowOpened(java.awt.event.WindowEvent e) {
+                	}
+
+                @Override
+                	public void windowClosing(java.awt.event.WindowEvent e) {
+                	}
+        	});
+
+                pack();
+        }
+
+//		setVisible(true);
+
+	 private JPanel getJPanel() {
+		jPanel.add(new JLabel("URL Form:"));
+		jPanel.add(printURL);
+		jPanel.add(buttonOK);
+
+		buttonOK.addActionListener(new ActionListener() {
+		@Override
+
+			public void actionPerformed(ActionEvent event) {
+			for (Enumeration<AbstractButton> buttons = group.getElements(); buttons.hasMoreElements();) {
+            			AbstractButton button = buttons.nextElement();
+
+            			if (button.isSelected()) {
+					JOptionPane.showMessageDialog(FieldEnumeration.this,"You selected: " + button.getText());
+             			   	System.out.println("button selected: " + button.getText());
+           			 }
+       			}
+			}
+		});
+			
+		return jPanel;
+	}
+	public HistoryReference getHistoryRef() {
+                return historyRef;
+        }
+
+        public void setHistoryRef(HistoryReference historyRef) {
+                this.historyRef = historyRef;
+		StringBuilder sb = new StringBuilder();
+                sb.append(historyRef.getURI().toString());
+                System.out.println("URL " + sb);
```

Comment:
Sorry I was unclear I guess. The `System.out.println` should be removed or replaced with use of `Logger`.